Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml

```diff
@@ -30,30 +33,62 @@
 
-                    <StackPanel Grid.Column="0" HorizontalAlignment="Left" Orientation="Horizontal">
-                        <Label Content="Page URL:" Style="{StaticResource $LabelStyle}" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                        <TextBox x:Name="xPageURLTextBox" Style="{StaticResource $TextBoxStyle}" HorizontalAlignment="Left" VerticalAlignment="Center" FontWeight="Bold" TextWrapping="NoWrap" Width="250" Margin="45,0,0,0"/>
-                        <UserControls:ucButton x:Name="xPageURLBtn" ButtonType="ImageButton" HorizontalAlignment="Center" ButtonFontImageSize="16" ButtonImageWidth="16" ButtonImageHeight="16" ButtonImageType="GoNext" Click="xPageURLBtn_Click" ToolTip="Go to Page"/>
-                    </StackPanel>
-
                     <agents:ucAgentControl x:Name="xAgentControlUC" Grid.Column="1" AgentStartedUCEvent="AgentStartedHandler" HorizontalAlignment="Right" Width="350" VerticalAlignment="Top" Margin="0,-5,0,0" ></agents:ucAgentControl>
+
+                    <TextBlock Grid.Column="0" Grid.Row="1" Margin="5,0,0,0" Text="Page Source" FontWeight="Bold" />
+                    <Grid Grid.Column="0" Grid.Row="2" >
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="150"></ColumnDefinition>
+                            <ColumnDefinition Width="Auto"></ColumnDefinition>
+                        </Grid.ColumnDefinitions>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+                        <StackPanel Grid.Column="0" Grid.Row="0" Margin="0,5,0,0" HorizontalAlignment="Left" >
+                            <RadioButton x:Name="xRDBPageURL" Height="20"  Grid.Column="0" Grid.Row="1" Tag="radPageURL" HorizontalAlignment="Left" Margin="10,5,0,0" >
+                                <RadioButton.Content>
+                                    <GingerCore:ucTextDicResource Text="Page URL"/>
+                                </RadioButton.Content>
+                            </RadioButton>
+                            <RadioButton x:Name="xRDBBF" Height="20" IsChecked="True"   Grid.Column="0" Grid.Row="1" Tag="radPageURL" HorizontalAlignment="Left" Margin="10,5,0,0" >
+                                <RadioButton.Content>
+                                    <GingerCore:ucTextDicResource Text="Business Flow"/>
+                                </RadioButton.Content>
+                            </RadioButton>           
+                        </StackPanel>
+
+                        <StackPanel Grid.Column="1" VerticalAlignment="Top" Grid.Row="0" Margin="5,2,0,0" Orientation="Horizontal" Visibility="{Binding ElementName=xRDBPageURL, Path=IsChecked, Converter={StaticResource visibilityConverter}}" >
+                            <Label Content="Page URL:" Style="{StaticResource $LabelStyle}" HorizontalAlignment="Left" VerticalAlignment="Center"/>
```

Comment:
for what we need the "Page URL:" label? we said we will show the TextBox and button and "URL" will be on Radio button only

Reply:
Done